App/components/Input/input.styled.tsx

```diff
@@ -0,0 +1,48 @@
+import styled, { css } from 'styled-components/native'
+
+interface PlaceholderLabelProps {
+  isFocused: boolean
+  isFilled: boolean
+}
+
+export const Container = styled.View`
+  width: 126px;
+  height: 50px;
+  margin-top: 40px;
+  border-bottom-width: 1px;
+  padding-right: 12px;
+  padding-left: 1px;
+  border-bottom-color: #dcdcdc;
+  display: flex;
+  flex-direction: row;
+`
+
+export const PlaceholderLabel = styled.Text<PlaceholderLabelProps>`
+  position: absolute;
+  left: 0;
+  color: #dcdcdc;
```

Comment:
Don't forget to use colors from theme ;)

Reply:
done